src/flows/shared/steps/OrganizeCollection/OrganizeCollection.tsx

```diff
@@ -1,55 +1,74 @@
-import { useEffect } from 'react';
+import { useCallback, useEffect } from 'react';
 import { WizardContext } from 'react-albus';
 import styled from 'styled-components';
 
 import CollectionNamingForm from './CollectionNamingForm';
 import CollectionEditor from './Editor/CollectionEditor';
 
 import { useWizardCallback } from 'contexts/wizard/WizardCallbackContext';
-import CollectionEditorProvider from 'contexts/collectionEditor/CollectionEditorContext';
+import { useStagedNftsState } from 'contexts/collectionEditor/CollectionEditorContext';
 import { useModal } from 'contexts/modal/ModalContext';
 import { useWizardId } from 'contexts/wizard/WizardDataProvider';
+import fetcher from 'contexts/swr/fetcher';
+import { EditModeNft } from 'types/Nft';
+import { CreateCollectionResponse } from 'types/Collection';
 
 type ConfigProps = {
   onNext: WizardContext['next'];
 };
 
 type Props = {
-  step?: any;
+  next?: any;
 };
 
+function mapStagedNftsToNftIds(stagedNfts: EditModeNft[]) {
+  return stagedNfts.map((stagedNft: EditModeNft) => stagedNft.nft.id);
+}
+
 function useWizardConfig({ onNext }: ConfigProps) {
   const wizardId = useWizardId();
   const { setOnNext } = useWizardCallback();
   const { showModal } = useModal();
+  const stagedNfts = useStagedNftsState();
+
+  const getStagedNfts = useCallback(() => stagedNfts, [stagedNfts]);
```

Comment:
I think there's a better way to do this ? I wanted to access the staged nfts returned by `useStagedNftsState()` from inside the `useEffect` without adding `stagedNfts` to the `useEffect`'s dependency array. 
The callback seemed like a good way to basically have a function that I could call in the `useEffect` do that. But the callback still gets updated every time the staged nfts change which I don't really want.

Ultimately I only care about what the value of `stagedNfts` is at the time onNext() is called  - when the user clicks "Save Collection"

any ideas?
@fbrcsftnr